### DISCUSSION TITLE
Multiple Domain MS 365 instance does not sync domain folders

### DISCUSSION THREAD

**Discussion by tremmers** [abraunegg/onedrive#3029] [General]
Environment: Microsoft 365 with multiple domains

I have multiple domains defined in my MS 365 instance, and the OneDrive -> My files folder for these have a Folder Icon with an arrow and a two-headed person icon. 
<img width="23" alt="image" src="https://github.com/user-attachments/assets/17ffc345-4281-4c46-822b-a6b1b3161897">
Software install works fine, and the files and folders w/o the funky folder icon sync, but none of the domain root directories appears on Unix. Enabling symlinks in the config file has no effect.

Is there a way to sync these folders to a Unix directory?

**Comment by abraunegg** [comment]
Have you read all the applicable documentation for syncing shared folders when using Business | Microsoft 365 ?

**Comment by tremmers** [comment]
If you don't know the answer to the question, then as a matter of etiquette, don't respond with cryptic nonsense.

**Comment by abraunegg** [comment]
Its not a cryptic response - its a very specific question - the answer to that question then will direct how next to help you.

The syncing of Shared Folders when using Business | Microsoft 365 accounts requires specific configuration which the documentation clearly articulates.

Please read:
* https://github.com/abraunegg/onedrive?tab=readme-ov-file#configure-onedrive-business-shared-items
* https://github.com/abraunegg/onedrive/blob/master/docs/business-shared-items.md

If you have not read these links then I suggest you start there.

**Comment by tremmers** [comment]
You are trying to run away from your first response. Negator, good buddy. Own it.

**Comment by tremmers** [comment]
I meant *nagatory*, but that reference is a bit dated. 

It looks like you own this repository. Just a suggestion, when people ask questions, just answer them. You gain nothing by being a smart-assed jerk. Just saying.
